pkg/server/util/grpccodesutil/checks.go

```diff
@@ -0,0 +1,14 @@
+package grpccodesutil
+
+import (
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func IsUnimplementedError(err error) bool {
```

Comment:
This helper can be implemented in one line (and is probably therefore unnecessary):

```
status.Code(err) == codes.Unimplemented
```